src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -0,0 +1,108 @@
+import com.onarandombox.MultiverseCore.commandtools.flags.CommandFlag;
+import com.onarandombox.MultiverseCore.commandtools.flags.CommandValueFlag;
+import com.onarandombox.MultiverseCore.commandtools.flags.CommandFlagGroup;
+import com.onarandombox.MultiverseCore.commandtools.flags.ParsedCommandFlags;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.WorldType;
+import org.bukkit.command.CommandException;
+import org.bukkit.plugin.Plugin;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class CreateCommand extends MultiverseCommand {
+
+    public CreateCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+
+        registerFlagGroup(CommandFlagGroup.builder("mvcreate")
+                .add(CommandValueFlag.builder("--seed", String.class)
+                        .addAlias("-s")
+                        .completion(() -> Collections.singleton(String.valueOf(new Random().nextLong())))
+                        .build())
+                .add(CommandValueFlag.builder("--generator", String.class)
+                        .addAlias("-g")
+                        .completion(() -> Arrays.stream(Bukkit.getServer().getPluginManager().getPlugins())
+                                .filter(Plugin::isEnabled)
+                                .filter(genplugin -> this.plugin.getUnsafeCallWrapper().wrap(
+                                        () -> genplugin.getDefaultWorldGenerator("world", ""),
+                                        genplugin.getName(),
+                                        "Get generator"
+                                ) != null)
+                                .map(genplugin -> genplugin.getDescription().getName())
+                                .collect(Collectors.toList()))
+                        .build())
+                .add(CommandValueFlag.builder("--world-type", WorldType.class)
+                        .addAlias("-t")
+                        .context((value) -> {
+                            try {
+                                return WorldType.valueOf(value.toUpperCase());
+                            } catch (IllegalArgumentException e) {
+                                throw new CommandException("Invalid world type: " + value);
+                            }
+                        })
+                        .completion(() -> {
+                            List<String> types = new ArrayList<>();
+                            for (WorldType type : WorldType.values()) {
+                                types.add(type.name().toLowerCase());
+                            }
+                            return types;
+                        })
+                        .build())
+                .add(CommandFlag.builder("--adjust-spawn")
+                        .addAlias("-n")
+                        .build())
+                .add(CommandFlag.builder("--no-structures")
+                        .addAlias("-a")
```

Comment:
Seams this should be `-n`

Reply:
Well not my fault, i need legacy support also ;-;; Clearly the previous mv devs use weird flags, but with autocomplete should be fine

```java
        boolean allowStructures = true;
        String structureString = CommandHandler.getFlag("-a", args);
        if (structureString != null) {
            allowStructures = Boolean.parseBoolean(structureString);
        }
        String typeString = CommandHandler.getFlag("-t", args);
        boolean useSpawnAdjust = true;
        for (String s : args) {
            if (s.equalsIgnoreCase("-n")) {
                useSpawnAdjust = false;
            }
        }
```